site/docs/v1/tech/core-concepts/scim.adoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: doc
+title: Search
+description: Search options within FusionAuth
+---
+
+:sectnumlevels: 0
```

Comment:
Should probably add  version flag somewhere here?

Reply:
done